packages/mask/src/plugins/ArtBlocks/SNSAdaptor/index.tsx

```diff
@@ -23,6 +25,15 @@ const sns: Plugin.SNSAdaptor.Definition = {
         const asset = getAssetInfoFromURL(collectibleUrl)
         return asset ? <Renderer chainId={asset.chain_id} projectId={asset.project_id} /> : null
     },
+    ApplicationEntries: [
+        {
+            category: 'dapp',
+            marketListSortingPriority: 15,
+            description: <Trans i18nKey="plugin_artblocks_description" />,
+            name: <Trans i18nKey="plugin_artblocks_name" />,
```

Comment:
```suggestion
            description: { fallback: "Some description", key: '__artblocks_description' },
            name: { fallback: 'Name', key: '__artblocks_name' },
```

Reply:
And to make this work, this plugin should provide the i18n resources in the plugin definition via `definition.i18n`. An example: packages/mask/src/plugins/MaskBox/base.ts line 16

Reply:
done